### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by CollRen** [582-41E-MA/projet-web-2-CollRen#35] [Announcements]
@claraquintela Tu peux essayer notre API en ligne ?

http://prodrdm.com/api/villes

**Comment by claraquintela** [comment]
Teste effectué avec sucess :)
Bravo!

Em qua., 26 de jun. de 2024 às 14:01, René de Montigny <
***@***.***> escreveu:

**Comment by claraquintela** [comment]
Dis moi quand je peux faire de test pour de vrai :)

Em qua., 26 de jun. de 2024 às 14:03, Clara Quintela <
***@***.***> escreveu:

**Comment by CollRen** [comment]
@claraquintela 

auth terminé!

Tu peux faire des tests avec les routes

- utilisateurs/login
- utilisateurs/logout
- utilisateurs/auth pour simplement vérifier si l'utilisateur actuel est connecté ou pas.

Utilisateur qui fonctionne:
    "nom_utilisateur": "Très Bon token",
    "mot_de_passe": "1234567"

Pour les tests en local, créé-toi un utilisateur avec un mot de passe et tu pourras ensuite tester les login et logout

**Reply by claraquintela** [reply]
Merci!
